File: SAML 2 IdP-Initiated Web Example for ASP.NET MVC/[VB]-SAML 2 IdP-Initiated Web Example for ASP.NET MVC/VB/Saml2IdPInitiatedMvc.SP/Controllers/HomeController.vb
Imports System.Web.Mvc

Namespace Saml2IdPInitiatedMvc.SP.Controllers
	<HandleError> _
	Public Class HomeController
		Inherits Controller
		<Authorize> _
		Public Function Index() As ActionResult
			ViewData("Message") = "Welcome to SAML SSO Idp-Initiated SP Example for ASP.NET MVC!"

			Return View()
		End Function
	End Class
End Namespace

File: SAML 2 IdP-Initiated Web Example for ASP.NET MVC/[C#]-SAML 2 IdP-Initiated Web Example for ASP.NET MVC/C#/Saml2IdPInitiatedMvc.SP/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace Saml2IdPInitiatedMvc.SP.Controllers
{
    [HandleError]
    public class HomeController : Controller
    {
        [Authorize]
        public ActionResult Index()
        {
            ViewData["Message"] = "Welcome to SAML SSO Idp-Initiated SP Example for ASP.NET MVC!";

            return View();
        }
    }
}
